prowler/providers/gcp/gcp_provider.py

```diff
@@ -88,7 +93,7 @@ def __init__(self, arguments):
             logger.critical(
                 "No Input Project IDs can be accessed via Google Credentials."
             )
-            sys.exit(1)
+            raise SystemExit
```

Comment:
Add a TODO here to replace this with our custom exception.

Reply:
Done